packages/shared/locales/en.json

```diff
@@ -586,7 +586,8 @@
         "restartInstall": "Restart to install",
         "calcMinutesRemaining": "Calculating minutes remaining...",
         "minutesRemaining": "{minutes, plural, one {1 minute remaining} other {# minutes remaining}}",
-        "copiedToClipboard": "Copied to clipboard"
+        "copiedToClipboard": "Copied to clipboard",
+        "accountsSynchronized": "Account synchronization complete."
```

Comment:
Smallest thing ever: should we add trailing full stop to every notification? Or just remove this one?

Reply:
Just remove it probably :)

Reply:
perfect, I will change it and merge ✔️